keylime/registrar_client.py

```diff
@@ -16,13 +16,15 @@
 
 logger = keylime_logging.init_logging('registrar_client')
 tls_cert_info = ()
+ca_cert = False
 tls_enabled = False
 api_version = keylime_api_version.current_version()
 
 
 def init_client_tls(section):
     global tls_cert_info
     global tls_enabled
+    global ca_cert
```

Comment:
Not to be resolved in this PR, but we are starting to have a lots of global state via global vars. This will be a pain later, and we should create some issue to try to remove them.

Reply:
I agree that we should start to remove global state where not strictly necessary.